src/utils/request.js

```diff
@@ -27,7 +27,7 @@ function filterStoppedTasks (objects) {
   let runningTasks = [];
   for(let i=0;i<objects.length;i++){
     let object = objects[i];
-    if(object.Status.State=="running" && object.DesiredState=="running") {
+    if( object.DesiredState=="running") { //object.Status.State=="running" &&
```

Comment:
Yes, If a containers DesiredState is running, but its actual  state is failed, I want it to appear in the UI with a red failed state,